src/Sylius/Bundle/AdminBundle/Action/Account/RenderResetPasswordPageAction.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Action\Account;
+
+use Sylius\Bundle\AdminBundle\Form\RequestPasswordResetType;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment;
+
+final class RenderResetPasswordPageAction
```

Comment:
IMO `RenderRequestPasswordResetPageAction` would better convey what page this controller actually refers to.